docs/src/components/Footer.tsx

```diff
@@ -0,0 +1,146 @@
+      <StyledContainer color="paletteCloudLight">
+        <StyledInner>
+          <Box display="flex" align="center" justify="between">
+            <StyledOutdent amount="spaceXXLarge">
+              <Inline align="center">
+                <StyledFooterLink to="/component-status">Component status</StyledFooterLink>
+                <div>·</div>
+                <StyledFooterLink to="/roadmap">Roadmap</StyledFooterLink>
+                <div>·</div>
+                <StyledFooterLink to="#">Road to React 1.0.0</StyledFooterLink>
+                <div>·</div>
+                <StyledFooterLink to="#">Changelog</StyledFooterLink>
+              </Inline>
+            </StyledOutdent>
+            <StyledOutdent amount="spaceXSmall">
+              <Inline align="center">
+                <StyledIconLink href="https://github.com/kiwicom/orbit">
+                  <GitHub />
+                </StyledIconLink>
+                <StyledIconLink href="https://spectrum.chat/orbit">
+                  <Spectrum />
+                </StyledIconLink>
+                <StyledIconLink href="https://twitter.com/OrbitKiwi">
+                  <Twitter />
+                </StyledIconLink>
+              </Inline>
+            </StyledOutdent>
+          </Box>
+        </StyledInner>
+      </StyledContainer>
+      <StyledContainer color="paletteCloudNormal">
+        <StyledInner>
+          <Inline align="center" justify="between">
+            <Box display="flex" align="center">
+              <Box padding={{ top: "large", bottom: "large" }}>
+                <Text as="div" weight="bold">
+                  Design tokens
+                </Text>
+                <Text as="div" type="secondary">
+                  v0.11.0
+                </Text>
+              </Box>
+              <Box padding={{ top: "large", bottom: "large", left: "XXLarge" }}>
+                <Text as="div" weight="bold">
+                  React components
+                </Text>
+                <Text as="div" type="secondary">
+                  v0.11.0
+                </Text>
+              </Box>
+              <Box
+                display="flex"
+                align="center"
+                justify="between"
+                padding={{ top: "large", bottom: "large", left: "XXLarge" }}
+              >
+                <Box shrink={0} padding={{ right: "XLarge" }}>
+                  <Text as="div" weight="bold">
+                    Developer mode
+                  </Text>
+                  <Text as="div" type="secondary">
+                    Opens components on the React tab by default.
+                  </Text>
+                </Box>
+                <Switch />
+              </Box>
+            </Box>
+            <Text align="right">
+              <Text as="div" weight="bold">
+                Built & maintained by Kiwi.com
+              </Text>
+              <Text as="div" type="secondary">
+                Open sourced with <Emoji>❤️</Emoji> for travel
```

Comment:
emoji here is totally fine :) but i think that it would be even better if it was a heart SVG icon in product color :)

Reply:
Great idea!